system/modules/task/actions/edit.php

```diff
@@ -206,12 +209,13 @@ function edit_POST($w) {
     }
     
     $task->fill($_POST['edit']);
+    $task->rate = abs($task->rate) == 0 ? NULL : $task->rate;
```

Comment:
Thanks Guy's,
I've updated the pull request.

regarding the removal of the abs function. validation for the field is
handled by the form, always going to be a positive two decimal float.
intval removes decimal places.


On Wed, Sep 14, 2016 at 10:58 AM, rlockerbie <notifications@github.com>
wrote:

> In system/modules/task/actions/edit.php
> <https://github.com/2pisoftware/cmfive/pull/213#discussion_r78671805>:
>
> > @@ -206,12 +209,13 @@ function edit_POST($w) {
> >      }
> >
> >      $task->fill($_POST['edit']);
> > +    $task->rate = abs($task->rate) == 0 ? NULL : $task->rate;
>
> Yup, just make changes on the same branch and push again, the changes will
> appear on the pull request automatically.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/2pisoftware/cmfive/pull/213/files/a0db7081daa98f7aea9915b52dc7151c39c34d60#r78671805>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIGsXIVEneknuWTmHgryIgHSMcYwhoSGks5qp0angaJpZM4J8Rd5>
> .
>